composer.json

```diff
@@ -25,6 +25,9 @@
         "aura/intl": "1.1.*",
         "psr/log": "1.0"
     },
+    "suggest": {
+      "ext-openssl": "To use Security::encrypt() or have secure CSRF token generation."
+    },
```

Comment:
Are you sure about that?
afaik, PHP7 has this built in - http://php.net/manual/en/function.random-bytes.php .